woocommerce-services.php

```diff
@@ -788,8 +788,16 @@ private function dismiss_tos_notice() {
 		 * @return bool
 		 */
 		private function can_accept_tos() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG && current_user_can( 'manage_options' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
```

Comment:
This check isn't needed, because `can_accept_tos` will only be called if `Jetpack` is installed and enabled (see https://github.com/Automattic/woocommerce-services/pull/1023/files#diff-57ae8afb227e356d71d37582f678a831R365 )